File: ios/Classes/model/Address.swift
//
//  Address.swift
//  contacts
//
//  Created by Maurits van Beusekom on 28/08/2018.
//

import Foundation

class Address : Codable {
    var label : String = ""
    var streetAddress : String = ""
    var city : String = ""
    var region : String = ""
    var country : String = ""
    var postalCode : String = ""
    var type : AddressType = AddressType.other
}

File: android/src/main/kotlin/com/baseflow/contactsplugin/models/Address.kt
package com.baseflow.contactsplugin.models

import com.baseflow.contactsplugin.data.AddressType

class Address {
    var type: AddressType = AddressType.OTHER
    var label: String = ""
    var streetAddress: String = ""
    var city: String = ""
    var region: String = ""
    var country: String = ""
    var postalCode: String = ""
}